docs/extensions/nodes.md

```diff
@@ -0,0 +1,30 @@
+# Adding Nodes or NodeTypes
+
+The whole programming-model is based on the Bitcoin-Core API. So we need Bitcoin Core Nodes or Elements Nodes or at least something which behaves like that. So for the Spektrum Integration, we made Extending the Node possible. This is a short description og how that has been done and which extension-points might be helpfull here.
+
+So To Create your own Node, Derive from `AbstractNode`., you have to specify the modules containing subclasses of `Device` in `service.py`:
```

Comment:
I can't follow here, did you really mean to refer to Device here? In Spectrum's `service.py` for example, there is no link to any Device related module.